controller_report_xls/reports/report_xls.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+#    Module Writen to OpenERP, Open Source Management Solution
+#    Copyright (C) Vauxoo (<http://www.vauxoo.com>).
+#    All Rights Reserved
+# #############Credits######################################################
+#    Coded by: Humberto Arocha <hbto@vauxoo.com>
+#    Coded by: Humberto Arocha <hbto@vauxoo.com>
+#    Planified by: Humberto Arocha <hbto@vauxoo.com>
+#    Audited by: Humberto Arocha <hbto@vauxoo.com>
+#############################################################################
+#    This program is free software: you can redistribute it and/or modify it
+#    under the terms of the GNU Affero General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or (at your
+#    option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+###############################################################################
+import base64
+import logging
+from io import BytesIO
+import lxml.html
+import xlwt
+from odoo import api, models
+
+from ..reports.xfstyle import css2excel
+
+# import simplejson
+
+_logger = logging.getLogger(__name__)
+
+
+def get_css_style(csstext, style):
```

Comment:
Is all this required?

Check how @WR-96 deals wit excel files using only Odoo requirements [here](https://git.vauxoo.com/vauxoo/apps/-/merge_requests/161).

On the other hand, if I'm not mistaking, excel is not strictly required, a CCSV would also do the trick (again, if I remember correctly).

Reply:
@luisg123v 

Yes, but this module already does the job, there is not need to code again, 

We agree at https://www.vauxoo.com/web?debug=1#id=29976&action=470&model=project.task&view_type=form&menu_id=317

that is less expensive in time use this module just migrating it than develop the csv/xsls implementation.

Reply:
and since is just for Lodi, I don't see any need of developing new improvements

Reply:
@hugho-ad thanks for clarifying, I agree, I though this was new code. Resolving.